### DISCUSSION TITLE
Copilot does not work since yesterday

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47629] [Copilot Conversations]
### Select Topic Area

I'm a paid user, when is it going to be fixed?

### Body

![image](https://user-images.githubusercontent.com/86713661/219318334-0111a1e9-12e9-4b02-9e6b-403afa7b6368.png)

![image](https://user-images.githubusercontent.com/86713661/219318358-c22d7633-6982-47f3-b29a-b683c1b0e749.png)

**Comment by tuves** [comment]
Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect. 

Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.

We sincerely apologize for any stress and/or frustration this has caused.

**Comment by tuves** [comment]
We became aware that Copilot was unavailable in several countries following our efforts to restore access to Ukraine, and on further investigation, we identified the cause as originating from a change we deployed to block Copilot in trade-restricted countries. To prevent this from happening again, we've updated our reporting dashboards to better account for error rates due to trade restrictions and are implementing more robust testing of the trade restriction logic.

**Reply by ghost** [reply]
"trade-restricted countries'
@tuves what Ukraine has to do anything with that?
